Storage/ThirdParty/SwiftData.swift

```diff
@@ -424,10 +436,13 @@ open class ConcreteSQLiteDBConnection: SQLiteDBConnection {
                 throw err
```

Comment:
Is this something we want to log? Or does this bubble up to a different point where we log it?